homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -67,7 +69,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     sensor = WazeTravelTime(name, origin, destination, region,
                             incl_filter, excl_filter, realtime)
 
-    add_entities([sensor])
+    add_entities([sensor], True)
```

Comment:
We shouldn't have included this change. We want to wait with updating until home assistant start event. We already have a listener for that, which will call update.

Reply:
so fix will be in 0.87.2?

Reply:
Please don't comment here. Let's keep the conversation in the issue.